src/store/getters/state-getters.ts

```diff
@@ -96,53 +96,53 @@ export const getEntityType = (state: StateIF): CorpTypeCd => {
 }
 
 /** Whether the entity is a Benefit Company. */
-export const isTypeBEN = (state: StateIF): boolean => {
+export const isBenefitCompany = (state: StateIF): boolean => {
   return (getEntityType(state) === CorpTypeCd.BENEFIT_COMPANY)
```

Comment:
The getter name is now consistent with the Corp Type Cd as per
https://github.com/bcgov/bcrs-shared-components/blob/main/src/modules/corp-type-module/corp-type-module.ts